test/slow/api_spec_slow.js

```diff
@@ -152,6 +152,12 @@ describe(`electron-forge API (with installer=${installer.substr(12)})`, () => {
       dir = path.resolve(os.tmpdir(), `electron-forge-test-${dirID}/electron-forge-test`);
       dirID += 1;
       await forge.init({ dir });
+
+      const packageJSON = await readPackageJSON(dir);
+      packageJSON.name = 'testApp';
```

Comment:
This is not a valid app name (could be causing the issues) has be all lower case